department-of-veterans-affairs/caseflow
# Renamed details components

Resolves #13871 

### Description
Renames details components in the hearings app.

### Acceptance Criteria

### Testing Plan
1. Select judge "BVAYELLOW": http://localhost:3000/test/users
1. Navigate to hearings:  http://localhost:3000/hearings/schedule
1. Choose any hearing
1. Click the "Edit hearing details" link
1. Ensure that all the functionality is unchanged
